.packit.yaml

```diff
@@ -0,0 +1,50 @@
+# See the documentation for more information:
+# https://packit.dev/docs/configuration/
+
+specfile_path: rubygem-smart_proxy_container_gateway.spec
+
+# add or remove files that should be synced
+files_to_sync:
+    - rubygem-smart_proxy_container_gateway.spec
+    - .packit.yaml
+
+# name in upstream package repository or registry (e.g. in PyPI)
+upstream_package_name: smart_proxy_container_gateway
+# downstream (Fedora) RPM package name
+downstream_package_name: rubygem-smart_proxy_container_gateway
+
+upstream_tag_template: "{version}"
+
+actions:
+  post-upstream-clone:
+    - "wget https://raw.githubusercontent.com/theforeman/foreman-packaging/rpm/develop/packages/plugins/rubygem-smart_proxy_container_gateway/rubygem-smart_proxy_container_gateway.spec -O rubygem-smart_proxy_container_gateway.spec"
+  get-current-version:
+    - ruby -rrubygems -e 'puts Gem::Specification::load(Dir.glob("*.gemspec").first).version'
+  create-archive:
+    - gem build smart_proxy_container_gateway.gemspec
+    - bash -c "ls -1t ./smart_proxy_container_gateway-*.gem | head -n 1"
+    #fix-spec-file:
+    #- 'bash -c "sed -i \"s/Source0:.*/Source0: ${PACKIT_PROJECT_ARCHIVE}/\" rubygem-smart_proxy_container_gateway.spec"'
+    #- bash -c 'sed -i "/global release/a%global nightly ${PACKIT_RPMSPEC_RELEASE:1}" rubygem-smart_proxy_container_gateway.spec'
```

Comment:
I think this is only needed for nightly packages and can be dropped safely.

Reply:
yeah, seems not needed :+1: